etl/steps/data/garden/growth/2024-01-04/gdp_historical.meta.yml

```diff
@@ -0,0 +1,128 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Economic Growth
+    processing_level: major
+
+    description_processing: |-
+      The data presented here from 1990 onwards is from the World Bank. Data before 1990 is constructed using the 1990 World Bank observation and applying the growth rates implied by the Maddison Project Database. Similarly, data before 1820 is built using the adjusted 1820 estimate calculated by the previous method and applying the growth rates implied by the Maddison Database.
```

Comment:
I think this could still be improved a bit - e.g., explain why you need to do this processing - ie. "Historical data is expressed in different units to World Bank, so to make it all consistent we can use the growth rates in our historical data and extrapolate back in time.. " or something along those lines. This is just to make it easy for everyone (and not just economists) to immediately understand the processing steps you've implemented but otherwise it all makes sense now. thanks for adding the comments